privacyidea/lib/tokens/pushtoken.py

```diff
@@ -217,6 +218,12 @@ def get_class_info(key=None, ret='all'):
                            'group': "PUSH",
                            'value': [gw.identifier for gw in gws]
                        },
+                       PUSH_ACTION.SSL_VERIFY: {
+                           'type': 'str',
```

Comment:
Why not use `bool` here?

Reply:
I thought about this first. But the defaul behaviour should be that SSL is verified. If we do a "bool" we can only have a policy, that it is NOT verified. This did not look nice. Also from a code perspective I was a bit lazy to check for a str in push token and check for a bool. So I only have string configs in the push token.

Reply:
Isn't [`is_true()`](https://github.com/privacyidea/privacyidea/blob/d1c7b37eca1f277658e68bedf9a256f206393c41/privacyidea/lib/utils.py#L725) for that?

Reply:
No. is_true comes from the different ways a database acutally stores a boolean value in the database. In this case we do not care about database and privacyIDEA.
In this case it is important how we communitcate with the smartphone App.